tgstation.dme

```diff
@@ -5265,6 +5265,7 @@
 #include "orbstation\code\jobs\department_security.dm"
 #include "orbstation\code\jobs\bartender\goblin_juice.dm"
 #include "orbstation\code\jobs\cargo\cadavers.dm"
+#include "orbstation\code\jobs\cargo\changed_packs.dm"
```

Comment:
oh my god do we really not have a better file for this? im so fucking mad. im screaming. im crying.

maybe someday this file and this change can go elsewhere but ARUGHGHHGHGH so mad about this

Reply:
we really didn't have ANYTHING that fits...

i can try to make a more general file to put it in? like just one for "packs" in general that don't need their own distinct file